### PR TITLE
Preview v5: Update package paths for `govuk-frontend@5`

### PR DESCRIPTION
**Pull request by colinrotherham**
PR to update package paths from [`govuk-frontend@5` **CHANGELOG.md**](https://github.com/alphagov/govuk-frontend/commit/cb64d8597792df45e67cde8f71d6f7f22c5d2011)

>#### Update package file paths
>
>In preparation for additional build targets, we've moved our package files into a directory called `dist`.

### CODE REVIEW

**Review by romaricpascal — COMMENTED** [review]
I know it's still in draft, but I added notes to update the name of the file we ask users to copy to ensure we don't miss it 😊

File: source/get-started/index.html.md.erb [hunk 1]
```diff
@@ -70,7 +70,7 @@ In your live application, we recommend [using an automated task or your build pi
     <script>document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');</script>
     ```
 
-2. Copy the `/node-modules/govuk-frontend/govuk/all.js` file to `<YOUR-JAVASCRIPT-FOLDER>/govuk.js`.
+2. Copy the `/node-modules/govuk-frontend/dist/govuk/all.js` file to `<YOUR-JAVASCRIPT-FOLDER>/govuk.js`.
```
[the remark this answers is not in the document]
Comment (colinrotherham) [reply source/get-started/index.html.md.erb]: ✅ Done

File: source/importing-css-assets-and-javascript/index.html.md.erb [hunk 2]
```diff
@@ -149,8 +149,8 @@ To import the JavaScript from GOV.UK Frontend, you can either:
 
 If you decide to add the JavaScript to your HTML, first either:
 
-- set up your routing so that requests for the JavaScript file are served from  `node_modules/govuk-frontend/govuk/all.js`
-- copy the `node_modules/govuk-frontend/govuk/all.js` file into your application
+- set up your routing so that requests for the JavaScript file are served from  `node_modules/govuk-frontend/dist/govuk/all.js`
```
[the remark this answers is not in the document]
Comment (colinrotherham) [reply source/importing-css-assets-and-javascript/index.html.md.erb]: ✅ Done

File: source/importing-css-assets-and-javascript/index.html.md.erb [hunk 3]
```diff
@@ -149,8 +149,8 @@ To import the JavaScript from GOV.UK Frontend, you can either:
...
+- copy the `node_modules/govuk-frontend/dist/govuk/all.js` file into your application
```
[the remark this answers is not in the document]
Comment (colinrotherham) [reply source/importing-css-assets-and-javascript/index.html.md.erb]: ✅ Done

**Comment by colinrotherham** [comment]
Thanks @romaricpascal, all updated

We'll convert to `<script type="module">` in another PR on top of this one

**Comment by colinrotherham** [comment]
Closing this one as it got combined with:

* https://github.com/alphagov/govuk-frontend-docs/pull/327

**Pull request closed without merging**